hack/local-up-cluster.sh

```diff
@@ -982,6 +1018,8 @@ EOF
 fi
 }
 
+
```

Comment:
Do we need the extra blank lines?

Reply:
I'm going to go out on a limb here and say no, probably not :smile: